src/resolvers/string.resolver.ts

```diff
@@ -23,3 +23,20 @@ export class StringResolver implements Resolver {
         return resolution;
     }
 }
+
+export class KeyResolver implements Resolver {
```

Comment:
Should validate that the key is a valid UInt64 string. Also, consider moving the resolver to a separate file.